deobfuscator-transformers/src/main/java/uwu/narumi/deobfuscator/core/other/impl/universal/number/CompareInstructionsTransformer.java

```diff
@@ -19,7 +20,8 @@ public class CompareInstructionsTransformer extends FramedInstructionsTransforme
   @Override
   protected Stream<AbstractInsnNode> getInstructionsStream(Stream<AbstractInsnNode> stream) {
     return stream
-        .filter(insn -> AsmMathHelper.isMathCompare(insn.getOpcode()));
+            .filter(Objects::nonNull)
```

Comment:
You don't need to check here if it is null as you already check it in `FramedInstructionsTransformer`